src/test/java/jenkins/branch/CustomOrganizationFolderDescriptorTest.java

```diff
@@ -76,35 +76,39 @@ public void removeIntegrationTestMocks() throws Exception {
     
     @Test
     public void noNavigatorNoFactoryInstalled() throws Exception {
-        assertEquals(1, ExtensionList.lookup(SCMNavigatorDescriptor.class).size());
-        assertEquals(SingleSCMNavigator.DescriptorImpl.class, ExtensionList.lookup(SCMNavigatorDescriptor.class).get(0).getClass());
-        assertEquals(0, ExtensionList.lookup(MultiBranchProjectFactoryDescriptor.class).size());
+        assertThat(ExtensionList.lookup(SCMNavigatorDescriptor.class),
+                contains(instanceOf(SingleSCMNavigator.DescriptorImpl.class)));
+        assertEquals(0, ExtensionList.lookup(MultiBranchProjectFactoryDescriptor.class).size()); // FIXME fails in PCT
         assertThat(newItemTypes(), containsInAnyOrder());
     }
 
     @Test
     public void someNavigatorNoFactoryInstalled() throws Exception {
-        assertEquals(2, ExtensionList.lookup(SCMNavigatorDescriptor.class).size());
-        assertEquals(0, ExtensionList.lookup(MultiBranchProjectFactoryDescriptor.class).size());
+        assertThat(ExtensionList.lookup(SCMNavigatorDescriptor.class),
+                containsInAnyOrder(instanceOf(SingleSCMNavigator.DescriptorImpl.class), instanceOf(MockNavigatorDescriptor.class)));
+        assertEquals(0, ExtensionList.lookup(MultiBranchProjectFactoryDescriptor.class).size()); // FIXME fails in PCT
         assertThat(newItemTypes(), containsInAnyOrder());
     }
     @TestExtension("someNavigatorNoFactoryInstalled")
     public static class SomeNavigatorNoFactoryInstalledDescriptor extends MockNavigatorDescriptor {}
 
     @Test
     public void noNavigatorSomeFactoryInstalled() throws Exception {
-        assertEquals(1, ExtensionList.lookup(SCMNavigatorDescriptor.class).size());
-        assertEquals(SingleSCMNavigator.DescriptorImpl.class, ExtensionList.lookup(SCMNavigatorDescriptor.class).get(0).getClass());
-        assertEquals(1, ExtensionList.lookup(MultiBranchProjectFactoryDescriptor.class).size());
+        assertThat(ExtensionList.lookup(SCMNavigatorDescriptor.class),
+                contains(instanceOf(SingleSCMNavigator.DescriptorImpl.class)));
+        assertThat(ExtensionList.lookup(MultiBranchProjectFactoryDescriptor.class),
+                contains(instanceOf(OrganizationFolderTest.MockFactoryDescriptor.class)));
```

Comment:
For example in this case I think you would say

```suggestion
                contains(ExtensionList.lookupSingleton(NoNavigatorSomeFactoryInstalledDescriptor.class)));
```

Reply:
done